src/Builders/PaymentMethodBuilder.php

```diff
@@ -167,6 +167,36 @@ public function get(): Collection
         }
     }
 
+    /**
+     * Set as default payment and/or of current subscriptions.
+     *
+     * @param  array<string, bool>  $payload
+     * @return bool
+     */
+    public function setAsDefault(array $payload = []): bool
+    {
+        try {
+            $endpoint = '';
+
+            if ($customer = $this->getOption('customer')) {
+                throw_unless(
+                    $methodId = $this->getOption('id'),
+                    MissingPaymentMethodException::class,
+                );
+
+                $endpoint = "customers/{$customer->id}/billing/methods/{$methodId}/primary";
+            }
+
+            resolve(Client::class)->patch($endpoint, $payload);
```

Comment:
You can build the payload here instead of passing it as a parameter, otherwise it's unclear to others what to expect in the payload.